mypy/build.py

```diff
@@ -188,10 +188,43 @@ def default_flush_errors(new_messages: List[str], is_serious: bool) -> None:
         raise
 
 
-def compute_lib_path(sources: List[BuildSource],
+SearchPaths = NamedTuple('SearchPaths',
```

Comment:
Could you please add a comment above documenting the fields and a typical usage.